packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -1,79 +1,83 @@
-
-    @media screen and (min-width: ${SITE_BREAKPOINTS[1]}) {
-      grid-template-columns: ${themeGet('sizes.sizeSidebar')} 1fr;
-    }
-  }
-`;
+const GlobalScrollBehaviourStyles = (scrollOffset = defaultScrollOffset): CSSObject => ({
+  html: {
+    scrollBehavior: 'smooth',
+    // compensate for the sticky topbar, this offset allows for the jump to links to keep the headings in view when you jump to the section
+    scrollPaddingTop: scrollOffset,
+  },
+});
 
 export const SiteBody: React.FC<React.PropsWithChildren> = ({children}) => {
   const {breakpointIndex} = useWindowSize();
   const themeObject = useTheme();
   const router = useRouter();
 
+  const [sidebarMobileCollapsed, setSidebarMobileCollapsed] = React.useState(true);
+
   /**
-   * The Tokens List page has a sticky filter when scrolled, which means that we need to set a
-   * CSS property called 'scrollPaddingTop' to adjust where the page scrolls to on jump links,
-   * so the content doesn't appear underneath the sticky filter.
-   *
-   * We have a global array, 'TOKEN_STICKY_FILTER_HEIGHT' that returns a value for the height
-   * of the filter (different breakpoints have different heights), and we use that, combined
-   * with router awareness of our current location, to determine the value to set for scrollPaddingTop.
-   *
-   * We also have a default value set to optimize the scroll jump so there's some space above the content.
+   * The tokens list page an extra sticky filter bar so the jump to scroll offset needs an extra offset.
+   * its the masthead height + the sticky filter height
    */
 
-  const defaultScrollOffset = 16;
   let scrollOffset = defaultScrollOffset;
 
   if (breakpointIndex !== undefined && TOKEN_LIST_PAGE_REGEX.test(router.pathname)) {
-    scrollOffset = TOKEN_STICKY_FILTER_HEIGHT[breakpointIndex] + defaultScrollOffset;
+    scrollOffset = `calc(${TOKEN_STICKY_FILTER_HEIGHT[breakpointIndex]}px + ${defaultScrollOffset})`;
   }
 
   return (
-    <Box display="flex" flexDirection="column" height="100vh">
-      <StylingGlobals styles={{...docSearchStyles({theme: themeObject}), ...docSearchVariable(themeObject)}} />
-      <SiteHeader />
-      <StyledSiteBody id="styled-site-body" css={{scrollPaddingTop: `${scrollOffset}px`}}>
-        {breakpointIndex === undefined || breakpointIndex > 1 ? <Sidebar /> : null}
-        <Box flex="1" minWidth="size0">
-          <main id={PASTE_DOCS_CONTENT_AREA}>{children}</main>
-          <SiteFooter />
-        </Box>
-      </StyledSiteBody>
-    </Box>
+    <>
+      <StylingGlobals
+        styles={{
+          ...GlobalScrollBehaviourStyles(scrollOffset),
+          ...docSearchStyles({theme: themeObject}),
+          ...docSearchVariable(themeObject),
+        }}
+      />
+      <SidebarDesktop
+        mainContentSkipLinkID={PASTE_DOCS_CONTENT_AREA}
+        sidebarNavigationSkipLinkID={PASTE_DOCS_SIDEBAR_NAV}
+        topbarSkipLinkID={PASTE_DOCS_TOPBAR}
+      />
+      <SidebarMobile
```

Comment:
question: Do we need two Sidebars still?

Reply:
OK, maybe you can help me on this one.

A single Sidebar can be collapsed (not visible) or expanded (visible). But the mobile sidebar has the opposite default collapsed state, and technically the desktop sidebar never collapses. 

I couldn't figure out a way to share a single sidebar and not get weird SSR behavior or weird collapsing behavior as you navigated around the site in desktop viewports. So my least worst just to get it working was a double sidebar. But this is also making testing hard